### PR TITLE
fix: wider code edit boxes

### PR DESCRIPTION
**Pull request by eMPee584**
- **fix: increase width of mongodb code edit fields by switching to QUERY_DYNAMIC_TEXT**
  Follow-up to
  18ef195f13cd2648cb82157e08792d256ff4f2f1 &
  1ecffb8272e45a83443797caf379c733f884eea2
  
  Fixes #15396
  

- **fix: reapply wider widths to firestore edit fields (#15365)**
  The fixes applied in
  1ecffb8272e45a83443797caf379c733f884eea2 (PR #15396)
  were accidentally overwritten by
  2f2cb42e0282551ad0a762bcfc61639cf6c613c5
  
  This restores those widget type changes.
  
  With this, all examples listed here should be resolved:
  https://appsmith.notion.site/The-problem-with-Form-Input-Widths-ba295cd8205b4809b7e8e97d75f3cc07

### CODE REVIEW

File: app/server/appsmith-plugins/firestorePlugin/src/main/resources/editor/setDocument.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
     {
       "label": "Timestamp Path",
       "configProperty": "actionConfiguration.formData.timestampValuePath.data",
-      "controlType": "QUERY_DYNAMIC_INPUT_TEXT",
```
Comment (sneha122) [inline app/server/appsmith-plugins/firestorePlugin/src/main/resources/editor/setDocument.json:26]: On app.appsmith.com today, I am not seeing the setDocument option, can you please let me know which option it corresponds to? @eMPee584

Comment (eMPee584) [reply app/server/appsmith-plugins/firestorePlugin/src/main/resources/editor/setDocument.json:26]: Well @Nikhil-Nandagopal and @sumitsum authored the file but according to https://github.com/appsmithorg/appsmith/blame/4e2f74ecbfca1d855e0fbc11109ba6d9b4067137/app/server/appsmith-plugins/firestorePlugin/src/main/resources/editor/root.json#L35 it is the "Upsert" option.. I don't have a firebase so can't test.

Comment (eMPee584) [reply app/server/appsmith-plugins/firestorePlugin/src/main/resources/editor/setDocument.json:26]: @sneha122 did you find it?..
these are very simple changes that had been applied before..

**Comment by eMPee584** [comment]
🎉

**Pull request closed without merging**